manifests/k/kite/kite/1.2021.527.0/kite.kite.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: kite.kite
+PackageVersion: 1.2021.527.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://kitedownloadss.b-cdn.net/windows/1.2021.527.0/KiteSetup.exe
```

Comment:
The `InstallerUrl` is unavailable again ...

I have some trouble surfing the Internet so I'm behind a proxy at the moment. I had switched my proxy server to different country or region many times, but none of them can help me successfully get the resource.

I think this package is still unavailable for `winget.exe`. Maybe we should abort this revert action ...

Reply:
Alrighty so its just me that is allowed to download it then 
![firefox_dNLLIqciTg](https://user-images.githubusercontent.com/15158490/155314019-f24e0522-5a91-4c2b-b915-aac4556bccc1.png)
```
Enter a pull request number: 47775

Manifest validation succeeded.
Found Kite [kite.kite] Version 1.2021.527.0
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://kitedownloadss.b-cdn.net/windows/1.2021.527.0/KiteSetup.exe
  ██████████████████████████████   405 MB /  405 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name        : Kite
Publisher   : Manhattan Engineering Inc
Version     :
ProductCode : Kite
Uninstall   : winget uninstall --id "Kite"

Name        : Microsoft Visual C++ 2015 Redistributable (x64) - 14.0.24212
Publisher   : Microsoft Corporation
Version     : 14.0.24212.0
ProductCode : {323dad84-0974-4d90-a1c1-e006c7fdbb7d}
Uninstall   : winget uninstall --id "{323dad84-0974-4d90-a1c1-e006c7fdbb7d}"

Press any key to continue . . .
```